recipes/lmdb/all/CMakeLists.txt

```diff
@@ -1,25 +1,24 @@
 cmake_minimum_required(VERSION 3.4)
-project(cmake_wrapper)
+project(lmdb LANGUAGES C)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup()
+include(GNUInstallDirs)
 
 # LMDB library:
 
 add_library(lmdb
-    source_subfolder/lmdb.h
-    source_subfolder/mdb.c
-    source_subfolder/midl.c
-    source_subfolder/midl.h
+    ${LMDB_SRC_DIR}/lmdb.h
+    ${LMDB_SRC_DIR}/mdb.c
+    ${LMDB_SRC_DIR}/midl.c
+    ${LMDB_SRC_DIR}/midl.h
 )
 
 set_target_properties(lmdb PROPERTIES
-    PUBLIC_HEADER source_subfolder/lmdb.h
+    PUBLIC_HEADER ${LMDB_SRC_DIR}/lmdb.h
     WINDOWS_EXPORT_ALL_SYMBOLS ON
 )
 
-include(FindThreads)
-target_link_libraries(lmdb PUBLIC ${CMAKE_THREAD_LIBS_INIT})
+find_package(Threads REQUIRED)
```

Comment:
Setting [THREADS_PREFER_PTHREAD_FLAG](https://cmake.org/cmake/help/latest/module/FindThreads.html#variable:THREADS_PREFER_PTHREAD_FLAG) to true before `find_package(Threads ...)` is recommend by CMake's documentation.

Reply:
I'm not sure it's really required. Most CMakeLists don't add this variable. It's required only if you have to pass -pthread to compiler.